### ISSUE TITLE
Default options override directive options

### ISSUE THREAD

**Issue by DarthBender** [mtr/angular-iscroll#32]
Hi, i'm trying to set `scrollbars: true` as default option in iScrollService and override it in a special case but i can't do it now. Is it normal behavior or options initialization in link function should be changed from

``` javascript
iScroll: angular.extend({}, scope.iscroll || {}, iScrollService.defaults.iScroll)
```

to

``` javascript
iScroll: angular.extend({},iScrollService.defaults.iScroll,  scope.iscroll || {})
```

**Comment by DarthBender** [comment]
I'm sorry, any updates on this? 
Thank you.

**Comment by mtr** [comment]
I'm sorry, I just haven't had the time yet because of other work.  I will look into this as soon as I get the chance.

**Comment by yehor-manzhula** [comment]
Hi! I've just made fix #34, please merge.

**Comment by mtr** [comment]
Thank you for your contribution.
